### PR TITLE
Removed duplicatate from platform.r ^ uls struct

### PR DESCRIPTION
**Pull request by DomShark**

### CODE REVIEW

File: src/bbu/chip8_raw.rs [hunk 1]
```diff
@@ -542,9 +542,6 @@ fn argcheck(a: &Option<Vec<String>>, i: usize) -> Vec<Chip8Arg> {
 }
 
 // TODO genericize for LE, BE
-struct Chip8Byte {
-    n: u8
```
Comment (amyipdev) [inline src/bbu/chip8_raw.rs:546]: You got it all and just it all, perfect

File: src/bbu/chip8_raw.rs [hunk 2]
```diff
@@ -542,9 +542,6 @@ fn argcheck(a: &Option<Vec<String>>, i: usize) -> Vec<Chip8Arg> {
...
-}
```
Comment (amyipdev) [inline src/bbu/chip8_raw.rs:545]: Thanks for keeping it down to one space

File: src/platform.rs [hunk 3]
```diff
@@ -53,7 +53,7 @@ impl Platform {
             // TODO: consider using some kind of lookup table?
             arch: match arch.to_lowercase().as_str() {
                 "chipeightraw" | "chip8raw" | "c8r" | "chip8r" => PlatformArch::ChipEightRaw,
-                "chipeight" | "chip8" | "c8" | "chip8" => PlatformArch::ChipEight,
+                "chipeight" | "chip8" | "c8"  => PlatformArch::ChipEight,
```
Comment (amyipdev) [inline src/platform.rs]: You accidentally added too (two) many spaces here, should just be one

**Review by amyipdev — CHANGES_REQUESTED** [review]
Please fix the spacing and then it's all good

File: src/platform.rs (hunk 3 above)
Comment (DomShark) [reply src/platform.rs]: Oh, my bad! Will fix it right now.

File: src/lexer.rs [hunk 4]
```diff
@@ -309,7 +309,7 @@ impl<T: crate::bbu::SymConv> Lexer<T> {
                 crate::platform::PlatformArch::ChipEight => {
                     LexOperation::Macro(crate::bbu::chip8::get_macro(i))
                 }
-                _ => panic!("not implemented yet")
+                // _ => panic!("not implemented yet")
```
Comment (amyipdev) [inline src/lexer.rs:312]: good catch, thanks

File: src/platform.rs [hunk 5]
```diff
@@ -53,7 +53,7 @@ impl Platform {
             // TODO: consider using some kind of lookup table?
             arch: match arch.to_lowercase().as_str() {
                 "chipeightraw" | "chip8raw" | "c8r" | "chip8r" => PlatformArch::ChipEightRaw,
-                "chipeight" | "chip8" | "c8"  => PlatformArch::ChipEight,
+                "chipeight" | "chip8" | "c8" => PlatformArch::ChipEight,
```
Comment (amyipdev) [inline src/platform.rs:56]: looks all good

**Review by amyipdev — APPROVED** [review]
current state lgtm

**Pull request merged**